basics/olympusgive.md

```diff
@@ -21,8 +21,10 @@ have partnered with.
 or organizations in your community.
 
 5. Once you have selected a recipient, determine the sOHM amount that you wish to
-pledge. The recipient will receive all the rebase rewards from the pledged tokens
-**indefinitely**, until all the principle is pulled out.
+pledge/deposit into the Give Pool for that recipient.
```

Comment:
```suggestion
pledge/deposit into the Give Vault for that recipient.
```

Reply:
I checked the figma, they call it Vault

Reply:
:-/  Ok - so how do I accept your change >_<